src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -56,22 +56,41 @@ public void storeMessage(Message message) {
    */
   public List<Message> getMessages(String recipient) {
     List<Message> messages = new ArrayList<>();
-
     Query query =
         new Query("Message")
             .setFilter(new Query.FilterPredicate("recipient", FilterOperator.EQUAL, recipient))
             .addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = datastore.prepare(query);
+    messages = message(query);
+    }
```

Comment:
This curly brace seem to have been left behind by mistake. You should remove it.